model/pom.xml

```diff
@@ -124,16 +134,6 @@
       <plugin>
         <groupId>org.flywaydb</groupId>
         <artifactId>flyway-maven-plugin</artifactId>
-
-        <executions>
-          <execution>
-            <phase>generate-sources</phase>
-            <goals>
-              <goal>migrate</goal>
-            </goals>
-          </execution>
-        </executions>
-
         <configuration>
```

Comment:
Rather than tying these goals into the standard build lifecycle via the executions tag, we're omitting that section so that they can only be invoked manually.